src/providers/bitpay-card/bitpay-card.ts

```diff
@@ -501,14 +505,24 @@ export class BitPayCardProvider {
     });
   }
 
-  public register() {
+  public async register() {
+    let countryCode;
+
+    try {
+      const res = await this.globalization.getLocaleName();
+      countryCode = res.value.split('-')[1];
+    } catch (error) {
+      this.logger.warn('Error retrieving country Code', error);
+    }
     this.isActive(isActive => {
       this.homeIntegrationsProvider.register({
         name: 'debitcard',
         title: 'BitPay Visa® Card',
         icon: 'assets/img/bitpay-card/icon-bitpay.svg',
         page: 'BitPayCardIntroPage',
-        show: !!this.configProvider.get().showIntegration['debitcard'],
+        show: !this.configProvider.get().isBitpayBannerStatusInitiallyConfigured
+          ? countryCode === 'US' || !this.platformProvider.isCordova
```

Comment:
I'm using proxy to simulate US territory, but the card banner still doesn't appear in a fresh install. Maybe it's something related to delay of globalization plugin (since it's an async function).